docs/api/session.md

```diff
@@ -698,6 +698,43 @@ session.fromPartition('some-partition').setPermissionCheckHandler((webContents,
 })
 ```
 
+#### `ses.setDisplayMediaRequestHandler(handler)`
+
+* `handler` Function | null
+  * `request` Object
+    * `frame` [WebFrameMain](web-frame-main.md) - frame that is requesting access to media.
+    * `securityOrigin` String - origin of the page making the request.
+    * `videoRequested` Boolean - true if the web content requested a video stream.
+    * `audioRequested` Boolean - true if the web content requested an audio stream.
+    * `userGesture` Boolean - whether a user gesture was active when this request was triggered.
+  * `callback` Function
+    * `streams` Object
+      * `video` Object | WebFrameMain (optional)
```

Comment:
Would it be simpler to document this as `DesktopCapturerSource | WebFrameMain`

Reply:
Maybe... it doesn't require the rest of the properties in `DesktopCapturerSource` though.

Reply:
I want it to be super clear where you get the `id` from 🤔 Maybe `Pick<DesktopCapturerSource, 'id' | 'name'>`.  Not sure if our docs parser supports that lol

Reply:
Perhaps I can make that clearer some other way!